examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Remove <code>diagrams[0]</code>  from the sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done.</p>